scripts/dashboard-gen.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
```

Comment:
Although the shebang says `sh` I believe that this is probably using bash or another shell on your system which supports arrays. The POSIX shell doesn't have this support and is the reason this script is throwing an error in CI. Changing the shell to `bash` is less portable but should fix the error.
```suggestion
#!/usr/bin/env bash
```